src/test/java/unit/CloneWorkspace.java

```diff
@@ -108,9 +109,7 @@ public void cloneWorkspace(TestInfo testInfo) throws Exception {
     workspaceCreator.login();
 
     // create a workspace
-    // `terra workspace create --format=json`
-    sourceWorkspace =
-        TestCommand.runAndParseCommandExpectSuccess(UFWorkspace.class, "workspace", "create");
+    sourceWorkspace = WorkspaceUtils.createWorkspace(workspaceCreator);
```

Comment:
for clone, the destination workspace won't be tracked automatically. We'll have to make sure the tests clean that up manually.

Reply:
Yeah, definitely. Those tests can call `CRLJanitor.registerWorkspaceForCleanup`, but we'll have to be sure we're actually doing that